src/App.js

```diff
@@ -1,14 +1,28 @@
-import React from "react";
+import React, { useState } from "react";
 import AppHeader from "./components/AppHeader";
 import AppContent from "./components/AppContent";
 
 import "./App.sass";
 
 function App() {
+  const [appCardState, setAppCardState] = useState({
+    header: "Header",
+    content: "Content",
+  });
+
+  function appCardStateHandler(data) {
+    setAppCardState((prevState) => {
+      return { ...prevState, ...data };
+    });
```

Comment:
Syntax sugar:  could be shortened:
`setAppCardState(prevState => ({ ...prevState, ...data }));`